src/middleware.ts

```diff
@@ -1,12 +1,35 @@
-import { clerkMiddleware } from '@clerk/nextjs/server'
+'use server'
+import 'server-only'
+import { clerkMiddleware, createRouteMatcher } from '@clerk/nextjs/server'
+const isMemberRoute = createRouteMatcher(['/fix-me/member(.*)'])
+const isResearcherRoute = createRouteMatcher(['/fix-me/researcher(.*)'])
```

Comment:
@therealmarv this is where we can check roles, I added the `fix-me` so it didn't match and stop everything from working